sites/main-site/src/content/blog/2025/retreat-2025.mdx

```diff
@@ -0,0 +1,150 @@
+Here are the key topics we discussed and the initiatives we're launching as a result.
+
+## Governance and teams
+
+As nf-core continues to grow, we need to distribute responsibilities more effectively. We discussed team functions and identified new members to join existing teams, with a particular focus on bringing in contributors from regions underrepresented in the community. We're also establishing an on-call process for core team members to ensure better response times and clearer ownership of ongoing tasks. Spreading the workload more evenly creates more opportunities for upskilling and community leadership across a truly global team, particularly important as we continue to scale.
+
+> We would like to welcome Kurayi to the outreach team, and Kübra, Arthur, and Jim to the maintainers team!
+
+One major outcome from these discussions was the creation of a new documentation team. Community feedback made it clear that our documentation is a significant pain point, and we need a dedicated focus to address this properly. You can read more about the team's mission and the plans for [Documentation: v2](#documentation-v2) below.
+
+```mermaid
+flowchart BT
+    core --> steering
+    infrastructure --> core
+    outreach --> core
+    maintainers --> core
+    safety --> steering
+    documentation --> core
+```
+
+## Documentation: v2
+
+Survey results and community feedback made it clear, our documentation needs work. People told us it's hard to find, sometimes unclear, and occasionally outdated.
+
+We're establishing [`#team-docs`](https://nfcore.slack.com/archives/C09H9L3R614) with a mandate to rebuild our docs from the ground up. This v2 docs project will implement a style guide, create a more effective structure, write new content while selectively migrating existing pages, and explore versioning (e.g., for component specifications) and automation options.
+
+The result will be documentation that's easier to find, more consistent and readable, creates a lower barrier to entry for new users, and provides more stability for experts. This will provide clear answers to common questions and free up our Slack community for more nuanced and/or technical questions and discussions.
+
+See [Documentation](https://nf-co.re/governance#documentation) for more information about the team.
+
+## Improving review and development practices
+
+Reviews are critical for maintaining quality, but we've heard your frustration. Overwhelmingly large PRs and multiple rounds of review can slow things down, and reviews often fall on the same people.
+
+We're making several key changes, including reducing the requirements for pipeline release reviews, from two reviewers to one (core team member or maintainer). We also want to start enforcing [conventional](https://www.conventionalcommits.org/en/v1.0.0/) PR titles to enable automated changelogs, more automation around the release,and add CI warnings for excessively large PRs.
```

Comment:
maybe make these as bullet points, so it is easier to read.

Reply:
Done 👍